src/debugger/appWorker.ts

```diff
@@ -60,9 +60,9 @@ export class SandboxedAppWorker {
 
     private static PROCESS_MESSAGE_INSIDE_SANDBOX = "onmessage({ data: postMessageArgument });";
 
-    constructor(sourcesStoragePath: string, debugAdapterPort: number, postReplyToApp: (message: any) => void, {
+    constructor(private packagerPort: number, sourcesStoragePath: string, debugAdapterPort: number, postReplyToApp: (message: any) => void, {
```

Comment:
I'm fine with either option. If the rest of the team agrees with not using this "private" keyword feature, I'll change the code to not use it. @hamiltonia @dlebu @MSLaguana @joshuaskelton What do you think?